sql/plan/foreign_key_editor.go

```diff
@@ -36,7 +36,7 @@ import (
 // "x4", which is in the fourth position (so 3 with zero-based indexed).
 type ChildParentMapping []int
 
-// ForeignKeyRefActionData contains the mapper, editor, and child to parent mapping for processing referential actions.
+// ForeignKeyRefActionData contains the mapper, editIter, and child to parent mapping for processing referential actions.
```

Comment:
Looks like you accidentally replaced occurrences of `editor` with `editIter` throughout the comments of the file.